CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 **Features**:
 
 - Use same keys for OTel span attributes and Sentry span data. ([#3457](https://github.com/getsentry/relay/pull/3457))
+- Properly identify meta web crawlers when filtering out web crawlers
```

Comment:
```suggestion
- Properly identify meta web crawlers when filtering out web crawlers. ([#4699](https://github.com/getsentry/relay/pull/4699))
```

nbd, but I assume you wanted to put it under bug fixes (judging by the PR title).